datalad/runner/tests/test_witless_runner.py

```diff
@@ -199,7 +199,13 @@ def pipe_data_received(self, fd, data):
         # value passed to protocol constructor
         value=b'5',
     )
-    eq_(res['stdout'], '5')
+    try:
+        eq_(res['stdout'], '5')
+    except AssertionError:
+        # TODO: remove when dropping support for python3.8
+        if res['stdout'] == '55' and sys.version_info[:2] == (3, 8):
```

Comment:
apparently not 3.8 specific -- happened for 3.9 https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=556819&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c10fa5f2-fdf6-5338-3bdb-c4bea7c23412